### ISSUE TITLE
Does not support scikit-learn=1.3.0

### ISSUE THREAD

**Issue by pokotylo** [TeamHG-Memex/eli5#425]
Cannot import eli5 after upgrading scikit-learn:

`import eli5`

```
File ".../python3.9/site-packages/eli5/sklearn/permutation_importance.py", line 7, in <module>
    from sklearn.utils.metaestimators import if_delegate_has_method
ImportError: cannot import name 'if_delegate_has_method' from 'sklearn.utils.metaestimators'
```

scikit-learn=1.3.0
eli5=0.13.0
python 3.9

**Comment by DamaKubu** [comment]
yeah same error 
```shell
ImportError: cannot import name 'if_delegate_has_method' from 'sklearn.utils.metaestimators' (/home/virtualbook/lib/python3.11/site-packages/sklearn/utils/metaestimators.py)
```

**Comment by SantaTitular** [comment]
Same here, any news?

**Comment by ElrondL** [comment]
Same here!!!

**Comment by wecounts** [comment]
same here！ wait for answers

**Comment by krol88** [comment]
Always the problem, so I can't use it...
It seems easy to fix why wouldn't you do it ?
Pleaaase :-)

**Comment by chocolatmint** [comment]
same for me!

**Comment by lopuhin** [comment]
hi, this is fixed in master branch of https://github.com/eli5-org/eli5 and we'll do a PyPI release soon

**Comment by lopuhin** [comment]
this is fixed in the latest release on PyPI: 0.14.0
